src/test/resources/org/broadinstitute/hellbender/tools/copynumber/utils/combine-segment-breakpoints-comparison.tsv.seg

```diff
@@ -0,0 +1,8 @@
+@HD	VN:1.5
+@SQ	SN:1	LN:2000000
+@RG	ID:GATKCopyNumber	SM:sample_gt
+CONTIG	START	END	Num_Probes	Segment_Mean	Segment_Call
```

Comment:
This test file still seems unnatural to me (see original comment directly below).  How would this be generated in the real world?  It has a CNV-collections-style header and a mix of "CONTIG/START/END" and CNV 1.0 column headers, so it seems like it would have to be formatted manually.

Is the tool be able to consume truth files in their original format?  If so, then the test files should reflect that.

Also, note the filename extension is .tsv.seg---probably this can be just .seg.

Reply:
Same goes for most of the test files below.

What is the spec for these tools?  I think they should be able to consume **strictly unmodified** CNV-collections files (namely, the output of CallCopyRatioSegments) and truth files (CNV 1.0-style seg files) along with the appropriate config files.  Requiring that the user go in and add additional headers or modify column names is something we should avoid.

Perhaps this is what the tools already do, but if so, it's not clear to me from looking at these test files.